src/mediasoupclient.cpp

```diff
@@ -16,6 +17,8 @@ namespace mediasoupclient
 
 		MSC_DEBUG("mediasoupclient v%s", Version().c_str());
 
+		webrtc::field_trial::InitFieldTrialsFromString("WebRTC-SupportVP9SVC/EnabledByFlag_3SL3TL/");
```

Comment:
I'm not sure this is the proper place to set those libwebrtc flags (maybe). @jmillan?

Reply:
Yes, this can be placed here.